### PR TITLE
app-text/wgetpaste: Version bump to v2.28

### PR DESCRIPTION
**Pull request by nvinson**

### CODE REVIEW

File: app-text/wgetpaste/wgetpaste-2.28.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit eutils
```
Comment (Whissi) [inline app-text/wgetpaste/wgetpaste-2.28.ebuild]: eutils eclass isn't used.

File: app-text/wgetpaste/metadata.xml [hunk 2]
```diff
@@ -2,7 +2,15 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<maintainer type="person">
+		<email>nvinson234@gmail.com</email>
```
Comment (Whissi) [inline app-text/wgetpaste/metadata.xml]: You added yourself before a Gentoo dev that makes you the primary maintainer according to https://wiki.gentoo.org/wiki/GLEP:67#Bug_assignment

Did the current maintainer (@maksbotan) acknowledged this change?

File: app-text/wgetpaste/metadata.xml [hunk 3]
```diff
@@ -2,7 +2,15 @@
...
+		<name>Nicholas Vinson</name>
+	</maintainer>
+	<maintainer type="person">
 		<email>maksbotan@gentoo.org</email>
 		<name>Maxim Koltsov</name>
 	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
```
Comment (Whissi) [inline app-text/wgetpaste/metadata.xml]: Same here, it is nice that you want to help. However the package has an active Gentoo developer. You cannot add the proxy-maintenance project by yourself.

**Comment by Whissi** [comment]
@nvinson: Why do you remove the previous version? Is it vulnerable or doesn't work anymore? Should v2.27 never go stable?

**Comment by maksbotan** [comment]
I've bumped and added you as proxy. Thanks for help.

**Pull request closed without merging**